shutil: tests/unit/test_cmdline.py::test_pretty_formatter fails with termcolor >= 2.1.0

Reproduced on 1.7.0 and 5f9b88a65a8c1e506885352bbd9b2a47900f5014, using Python 3.11.1. The test passes with `termcolor < 2.1` but fails with 2.1.0 and 2.1.1.

```pytb
________________________________________________________ test_pretty_formatter ________________________________________________________

    def test_pretty_formatter():
        f = cmdline.PrettyFormatter()
        f.title('A Title')
        f.hr()
        f.p('A Paragraph', 'red')
>       assert f.buffer == [
            '\x1b[1m\x1b[34m  A Title\x1b[0m',
            '\x1b[1m\x1b[34m--------------------------------------------------------------------------------\x1b[0m',
            '\x1b[31mA Paragraph\x1b[0m'
            ]
E       AssertionError: assert ['  A Title', '--------------------------------------------------------------------------------', 'A Paragraph'] == ['\x1b[1m\x1b[34m  A Title\x1b[0m', '\x1b[1m\x1b[34m--------------------------------------------------------------------------------\x1b[0m', '\x1b[31mA Paragraph\x1b[0m']
E         At index 0 diff: '  A Title' != '\x1b[1m\x1b[34m  A Title\x1b[0m'
E         Full diff:
E           [
E         -  '\x1b[1m\x1b[34m  A Title\x1b[0m',
E         +  '  A Title',
E         -  '\x1b[1m\x1b[34m--------------------------------------------------------------------------------\x1b[0m',
E         ?   ---------------                                                                                -------
E         +  '--------------------------------------------------------------------------------',
E         -  '\x1b[31mA Paragraph\x1b[0m',
E         +  'A Paragraph',
E           ]

/tmp/pytest-plugins/pytest-shutil/tests/unit/test_cmdline.py:18: AssertionError
```

--- a/pytest-shutil/pytest_shutil/cmdline.py
+++ b/pytest-shutil/pytest_shutil/cmdline.py
@@ -52,6 +52,8 @@ def chdir(dirname):
 class PrettyFormatter(object):
     def __init__(self, color=True):
         from termcolor import colored
+        if color is False:
+            os.environ["FORCE_COLOR"] = "true"
         self.color = color
         self.colored = colored
         self.buffer = []

--- a/pytest-shutil/tests/unit/test_cmdline.py
+++ b/pytest-shutil/tests/unit/test_cmdline.py
@@ -10,7 +10,8 @@ def test_umask(workspace):
         f.touch()
     assert (f.stat().st_mode & 0o777) == 0o464
 
-def test_pretty_formatter():
+def test_pretty_formatter(monkeypatch):
+    monkeypatch.setenv("FORCE_COLOR", "1")
     f = cmdline.PrettyFormatter()
     f.title('A Title')
     f.hr()

